### ISSUE TITLE
[Bug] Variables in calc() cannot be compiled

### ISSUE THREAD

**Issue by uncaught** [Thream/styled-jsx-plugin-sass#135]
## Steps To Reproduce

1. Use a variable inside a calc() function

## The current behavior
A compile error `Error: Expected "(" or ".".` is thrown:

![image](https://user-images.githubusercontent.com/10322643/167083730-4f3a4238-5c64-4af8-a760-955d0e34d86d.png)

## The expected behavior
No compile error.

-----

I have created a minimal test repository with two calc cases:

```bash
git clone git@github.com:uncaught/styled-jsx-sass-var-in-calc-issue.git
cd styled-jsx-sass-var-in-calc-issue/
npm install
npm start
```

-----

These usages of variables in calc() work with `node-sass` and `styled-jsx-plugin-sass`.

**Comment by tudorc07** [comment]
Did you find any solution/workaround for this?

**Comment by uncaught** [comment]
No, unfortunately not. I had to downgrade back to [node-sass](https://www.npmjs.com/package/node-sass) and [styled-jsx-plugin-sass](https://www.npmjs.com/package/styled-jsx-plugin-sass).

**Comment by tudorc07** [comment]
You can use CSS variables to workaround this issue
